napari/components/dims.py

```diff
@@ -408,9 +585,62 @@ def _roll(self):
         order[nsteps > 1] = np.roll(order[nsteps > 1], 1)
         self.order = order.tolist()
 
+    def _sanitize_input(self, axis, value, value_is_sequence=False):
+        """
+        Ensure that axis is and value are the same length, that axis are not
```

Comment:
Less sure about this one

```suggestion
        Ensure that axis and value are the same length, that axes are not
```